doc/neps/nep-0055-string_dtype.rst

```diff
@@ -372,12 +377,12 @@ DType using the DType class.
 We will also extend the ``NPY_TYPES`` enum in the C API with an ``NPY_VSTRING``
 entry (there is already an ``NPY_STRING`` entry). This should not interfere with
 legacy user-defined DTypes since the integer type numbers for these data types
-begin at 256, so in principle there is still room for hundreds more builtin
+begin at 256. In principle there is still room for hundreds more builtin
 DTypes in the integer range available in the ``NPY_TYPES`` enum.
```

Comment:
A bit random:  Happy if we do also happy if not because it turns out to be a burden in practice.  (We should make the actual DTypes more accessible anyway.)